CodeEditModules/Modules/StatusBar/src/Model/StatusBarModel.swift

```diff
@@ -39,17 +64,25 @@ public class StatusBarModel: ObservableObject {
     public var currentCol: Int = 1 // Implementation missing
 
     /// Returns true when the drawer is visible
-    @Published
+    @AppStorage(StatusBarStorageKey.isExpanded.rawValue)
```

Comment:
I don't think AppStorage is a good way to go. What if a user has more than one workspace open? Then it would change in all workspaces.

Reply:
I don't think we need persistence for such things. But that's my opinion.